ibis/omniscidb/tests/test_client.py

```diff
@@ -100,6 +100,148 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+def test_add_column_zero(con):
+    table_name = 'my_table'
+
+    con.drop_table(table_name, force=True)
+
+    schema = ibis.schema([('a', 'float'), ('b', 'int8')])
+
+    con.create_table(table_name, schema=schema)
+
+    dict_cols_with_types = {}
+
+    isException = False
+    try:
+        con.add_column(table_name, dict_cols_with_types)
+    except com.IbisInputError:
+        isException = True
+    finally:
+        con.drop_table(table_name)
+
+    if not isException:
+        assert False
+
+
+def test_add_column_one(con):
+    table_name = 'my_table'
+
+    con.drop_table(table_name, force=True)
+
+    schema = ibis.schema([('a', 'float'), ('b', 'int8')])
+
+    con.create_table(table_name, schema=schema)
+
+    dict_cols_with_types = {'c': 'DOUBLE'}
+    con.add_column(table_name, dict_cols_with_types)
+
+    schema_for_check = ibis.schema(
+        [('a', 'float'), ('b', 'int8'), ('c', 'double')]
+    )
+
+    try:
+        t = con.table(table_name)
+
+        for k, i_type in t.schema().items():
+            assert schema_for_check[k] == i_type
+    finally:
+        con.drop_table(table_name)
+
+
+def test_add_column_several(con):
+    table_name = 'my_table'
+
+    con.drop_table(table_name, force=True)
+
+    schema = ibis.schema([('a', 'float'), ('b', 'int8')])
+
+    con.create_table(table_name, schema=schema)
+
+    dict_cols_with_types = {'c': 'DOUBLE', 'd': 'TEXT'}
```

Comment:
I think you need to test adding columns with more types, at least one with geo type.

Reply:
Done